pool/transaction.go

```diff
@@ -77,8 +78,8 @@ func (tx *Transaction) IsClaimTx(l2BridgeAddr common.Address) bool {
 		return false
 	}
 
-	if *tx.To() == l2BridgeAddr { // &&
-		// strings.HasPrefix("0x"+common.Bytes2Hex(tx.Data()), bridgeClaimMethodSignature)  {
+	if *tx.To() == l2BridgeAddr &&
+		strings.HasPrefix("0x"+common.Bytes2Hex(tx.Data()), bridgeClaimMethodSignature) {
```

Comment:
Not sure about this change, bcs it was commented in this commit - https://github.com/0xPolygonHermez/zkevm-node/commit/4e03c05d6900b7937cef9a6b40ae2b99bb4f8aa9#diff-b87b703deb53896d58cb942d1585ea5a40f705af35f3e12e92c8b3e2548a092bR81. @arnaubennassar what do you thing?

Reply:
I've already commented to him. A claimable tx goes to the l2bridge smc and to the method claim (it is filtered by the signature). If we remove that check we will consider deposits as claimables (gas 0)